umbral/_pre.py

```diff
@@ -113,8 +108,15 @@ def assess_cfrag_correctness(cfrag: 'CapsuleFrag', capsule: 'Capsule') -> bool:
     xcoord = cfrag._point_xcoord
     kfrag_id = cfrag._kfrag_id
 
-    kfrag_validity_message = bytes().join(
-        bytes(material) for material in (kfrag_id, delegating_point, receiving_point, u1, ni, xcoord))
+    if delegating_pubkey is None:
+        delegating_pubkey = bytes(UmbralPublicKey.expected_bytes_length())
+
+    if receiving_pubkey is None:
+        receiving_pubkey = bytes(UmbralPublicKey.expected_bytes_length())
```

Comment:
While this totally works, I hate using it because it makes no sense.

Instead, I prefer `b'\x00' * UmbralPublicKey.expected_bytes_length()`

Reply:
Yeah, I agree it makes no sense, while your approach is crystalline. I'll make the changes.